community/assessments/projects/k8gb/self-assessment.md

```diff
@@ -0,0 +1,221 @@
+  * [Deployment pipeline](#deployment-pipeline)
+  * [Communication channels](#communication-channels)
+* [Security issue resolution](#security-issue-resolution)
+  * [Responsible disclosure practice](#responsible-disclosure-practice)
+  * [Incident response](#incident-response)
+* [Appendix](#appendix)
+
+## Metadata
+
+### Security links
+|   |  |
+| - | - |
+| Software | https://github.com/k8gb-io/k8gb  |
+| Security Provider | No, k8gb is not a security provider. It relies on Kubernetes' built-in security mechanisms (e.g., RBAC, network policies) and does not implement standalone security features. |
+| Languages | Golang |
+| SBOM | https://github.com/k8gb-io/k8gb/releases/download/v0.14.0/k8gb_0.14.0_linux_amd64.tar.gz.sbom.json |
+| Security Insights | https://github.com/k8gb-io/k8gb/blob/master/SECURITY-INSIGHTS.yml |
+| Security File | https://github.com/k8gb-io/k8gb/blob/master/SECURITY.md |
+| Cosign pub-key | https://github.com/k8gb-io/k8gb/blob/master/cosign.pub |
+| | |
+
+### Intended use
+
+To increase the software supply chain security, we encourage our users to consume k8gb container images with Kyverno's admission webhook
+([/policy](https://kyverno.io/docs/writing-policies/verify-images/sigstore/#verifying-image-signatures)) that will ensure that
+images are signed and nobody had tempered with them. Our public key that can be used to verify this is in the root or our repository.
+
+## Overview
+
+k8gb is implemented using the Kubernetes operator pattern with a single CRD to enable Global Load Balancing. k8gb provides independent GSLB capability to any Ingress or Service without a dedicated management cluster, instead relying on timeproof DNS. k8gb has no single point of failure, and uses Kubernetes native application health checks (such as liveness and readiness probes) to update DNS to aid in load balancing decisions.
+
+### Background
+
+Global enterprises moving to the cloud need a global load balancer to make decisions based on Kubernetes services. Proprietary software and vendors exist, but these are expensive, complex, and not cloud native. Some require dedicated hardware and run outside Kubernetes.
+
+k8gb is a vendor-neutral, CNCF Sandbox project. It is the only cloud native Kubernetes Global Load Balancer. k8gb does not require any special software or hardware - it relies only on OSS / CNCF projects, and fits with existing Kubernetes workflows like: GitOps, Kustomize, and Helm.
+
+### Actors
+
+The individual parts of k8gb that interact to provide the desired functionality.
+
+- CoreDNS
+  - Role: Embedded [custom CoreDNS](https://github.com/k8gb-io/coredns-crd-plugin) to serve DNS requests.
+  - Isolation: runs as its own Pod
+- ExternalDNS
+  - Role: Integrated ExternalDNS to automate zone delegation configuration.  
+  - Isolation: runs as its own Pod
+- k8gb Controller
+  - Role: Coordinates logic according to the GSLB strategy.
+  - Isolation: runs as its own Pod
+
+### Actions
```

Comment:
This section would need to be fleshed out for a joint assessment.

Reply:
Thanks @JustinCappos - noted. For now, we're focusing on the self-assessment https://github.com/cncf/tag-security/issues/1441#issuecomment-2621683616 👍